### DISCUSSION TITLE
2 improvements for beginners

### DISCUSSION THREAD

**Discussion by Adrianotiger** [nextcloud/all-in-one#5110] [Ideas] [answered]
Hi,
I discovered Nextcloud 2 weeks ago. I ordered a PC and installed this AIO on Windows.
You wrote fantastic tutorials and even if I didn't know what a docker is, I was able to install it without any problems and it was working after just some minutes. Thank you very much!

But then my first problem: 
- I was thinking that I was setting the installation path of nextcloud but it was the data folder (I want it to an external ssd, so I can backup it easly). I tried to change some settings, but I was not able to do it without any error on rebooting the docker - At the end, I removed everything and installed AIO again.
- Would it be possible to integrate this feature in the Nextcloud AIO interface?

Now my second problem:
- I have a free dyndns and I am afraid that they could just remove my dns entry. If I read the tutorials, there isn't an uncomplicated way to change it, right? I need to change it in the apache server, the config.php and files on other images? I just have a home server with 4-5 users, so I can change the client urls.
- It would also be nice if there is such a feature in AIO interface :D

As said, I am just a beginner. But (if I understand it) the AIO interface has full access to the docker images. Or also on the OS (to create new cron jobs - or scheduled tasks)?

Thank you again for this fantastic tool!

**Comment by szaimen** [comment, marked as answer]
Hi,

> I discovered Nextcloud 2 weeks ago. I ordered a PC and installed this AIO on Windows.
> You wrote fantastic tutorials and even if I didn't know what a docker is, I was able to install it without any problems and it was working after just some minutes. Thank you very much!

Thanks! It is always great to receive positive feedback :)



> * I was thinking that I was setting the installation path of nextcloud but it was the data folder (I want it to an external ssd, so I can backup it easly). I tried to change some settings, but I was not able to do it without any error on rebooting the docker - At the end, I removed everything and installed AIO again.
> * Would it be possible to integrate this feature in the Nextcloud AIO interface?

We consider changing the datadir as an advanced setting. That is why we currently have no plans to make this adjustable via the aio interface.



> * I have a free dyndns and I am afraid that they could just remove my dns entry. If I read the tutorials, there isn't an uncomplicated way to change it, right? I need to change it in the apache server, the config.php and files on other images? I just have a home server with 4-5 users, so I can change the client urls.

You can change the domain by following https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-domain
